platform-tests/acceptance/aws_endpoint_check_test.go

```diff
@@ -0,0 +1,130 @@
+package acceptance_test
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	. "github.com/onsi/gomega/gexec"
+
+	"github.com/cloudfoundry/cf-test-helpers/cf"
+	"github.com/cloudfoundry/cf-test-helpers/generator"
+)
+
+type instanceIdentity struct {
+	Region string `json:"region"`
+}
+
+func GetAWSRegion() (string, error) {
+	resp, err := http.Get("http://169.254.169.254/latest/dynamic/instance-identity/document")
+	if err != nil {
+		return "", err
+	}
+	defer resp.Body.Close()
+
+	body, err := ioutil.ReadAll(resp.Body)
+	if err != nil {
+		return "", err
+	}
+
+	var identity instanceIdentity
+	if err := json.Unmarshal(body, &identity); err != nil {
+		return "", err
+	}
+
+	return identity.Region, nil
+}
+
+func AppCurl(appName string, url string) string {
+
+	cfSSHCommand := cf.Cf("ssh", appName, "-c", fmt.Sprintf("curl -s -o /dev/null -I -w \"%%{http_code}\" %s", url))
+	session := cfSSHCommand.Wait(testConfig.DefaultTimeoutDuration())
+	Expect(session).To(Exit(0))
+	return string(session.Out.Contents())
+}
+
+var _ = Describe("AWS Endpoint Check", Ordered, func() {
+	Context("when calling aws endpoints", func() {
+
+		var (
+			appName string
+			region  string
+			err     error
+		)
+
+		BeforeAll(func() {
+			region, err = GetAWSRegion()
+			Expect(err).NotTo(HaveOccurred())
+
+			appName = generator.PrefixedRandomName(testConfig.GetNamePrefix(), "APP")
+			Expect(cf.Cf(
+				"push", appName,
+				"--no-start",
+				"-m", DEFAULT_MEMORY_LIMIT,
+				"-p", "../example-apps/simple-python-app",
+				"-b", "python_buildpack",
+				"-c", "python hello.py",
+			).Wait(testConfig.CfPushTimeoutDuration())).To(Exit(0))
+			Expect(cf.Cf("start", appName).Wait(testConfig.CfPushTimeoutDuration())).To(Exit(0))
+		})
+
+		AfterAll(func() {
+			Expect(cf.Cf("delete", appName, "-f").Wait(testConfig.DefaultTimeoutDuration())).To(Exit(0))
+		})
+
+		It("should connect to secrets manager endpoint", func() {
```

Comment:
These could be easier to read as [a table test](https://onsi.github.io/ginkgo/#table-specs)? Thoughts?

Reply:
Yeah not a bad idea. Let me give it a try.